docs/setup_instructions/mac_installation_instructions.md

```diff
@@ -126,18 +126,19 @@ Once the installation process is completed, you can now safely delete (eject) th
 
 ### Installing necessary R packages
 
-For this workshop, you will need to have the following R packages installed: 
+For this workshop, you will need to have the following R packages installed:
 
 * `tidyverse`
 * `rmarkdown`
 * `optparse`
 * `renv`
+* `here`
```

Comment:
You might want to match the list of packages here to the order that they are installed below.